generators/client/templates/react/_package.json

```diff
@@ -29,12 +29,14 @@ limitations under the License.
     "availity-reactstrap-validation": "2.0.0-alpha.1",
     "axios": "0.17.0",
     "bootstrap": "4.0.0-beta.2",
+    "font-awesome": "4.7.0",
```

Comment:
we already have fontawesome from "react-icons" so use that and remove this please